gp-includes/formats/format_pomo.php

```diff
@@ -31,11 +32,36 @@ public function print_exported_file( $project, $locale, $translation_set, $entri
 		foreach( $entries as $entry ) {
 			$po->add_entry( $entry );
 		}
-		$po->set_header( 'Project-Id-Version', $project->name );
 
-		// TODO: include parent project's names in the comment
-		$po->comments_before_headers .= "Translation of {$project->name} in {$locale->english_name}\n";
-		$po->comments_before_headers .= "This file is distributed under the same license as the {$project->name} package.\n";
+		$current = $project;
+		$project_tree = array();
+		$project_tree[] = $current->name;
+
+		while ( $current->parent_project_id > 0 ) {
+			$current = GP::$project->get( $project->parent_project_id );
+			$project_tree[] = $current->name;
+		}
```

Comment:
We can always move it later if we come up with a use, however we should probably look at the project data to include some editable fields, like an application name (in addition to the project name), application version, etc. to better handle these kinds of cases.

If we were to add it to the project object I'd think something like `GP_Project::get_project_hiarchy( $separator = ' - ' ) { }` would be more appropriate, since as you pointed out in another comment, it's not just the "name", its more of a string representation of the project path in GlotPress.